topics/metagenomics/tutorials/metagenomics-binning/turorial.md

```diff
@@ -0,0 +1,418 @@
+- **Horizontal gene transfer**: Horizontal gene transfer (HGT) can complicate metagenomic binning, as it can introduce genetic material from one organism into another.
+- **Chimeric sequences**: Sequences that are the result of sequencing errors or contamination can lead to chimeric sequences, which can make it difficult to accurately bin reads.
+- **Strain variation**: Organisms within a species can exhibit significant genetic variation, which can make it difficult to distinguish between different strains in a metagenomic sample.
+
+<!-- > <details-title>Strategies for assembly</details-title>
+> For assembly, there are 3 main strategies:
+> 1. Greedy extension
+> 2. Overlap Layout Consensus
+> 3. De Bruijn graphs. The following figure illustrates these strategies in brief.
+>
+> ![Image shows greedy extention, overlap layout consensus, and de Brujin graphs assembly algorithms](./images/assembly-algorithms.png "Assembly algorithms. Image from {% cite carpentries %}"){:width="70%"}
+>
+> The nice paper {% cite miller2010 %} on assemblers based on these algorithms will help you to better understand how they work.
+{: .details} -->
+
+There are plenty of computational tools to perform metafenomics binning. Some of the most widely used include:
+
+- **MaxBin**: A popular de novo binning algorithm that uses a combination of sequence features and marker genes to cluster contigs into genome bins.
+- **MetaBAT**: Another widely used de novo binning algorithm that employs a hierarchical clustering approach based on tetranucleotide frequency and coverage information.
+- **CONCOCT**: A de novo binning tool that uses a clustering algorithm based on sequence composition and coverage information to group contigs into genome bins.
+- **MyCC**: A reference-based binning tool that uses sequence alignment to identify contigs belonging to the same genome or taxonomic group.
+- **GroopM**: A hybrid binning tool that combines reference-based and de novo approaches to achieve high binning accuracy.
+- **MetaWRAP**: A comprehensive metagenomic analysis pipeline that includes various modules for quality control, assembly, binning, and annotation.
+- **Anvi'o**: A platform for visualizing and analyzing metagenomic data, including features for binning, annotation, and comparative genomics.
+
+**In this tutorial, we will learn how to run metagenomic binning tools and evaluate the quality of the results**. In order to do that, we will use data from the study: [Temporal shotgun metagenomic dissection of the coffee fermentation ecosystem](https://www.ebi.ac.uk/metagenomics/studies/MGYS00005630#overview) and MetaBAT2 algorithm. For an in-depth analysis of the structure and functions of the coffee microbiome, a temporal shotgun metagenomic study (six time points) was performed. The six samples have been sequenced with Illumina MiSeq utilizing whole genome sequencing.
```

Comment:
Why did you use MetaBAT here?

Reply:
Reasons for using MetaBAT2 are now included